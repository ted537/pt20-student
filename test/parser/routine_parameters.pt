using input, output;
fun flag(number : like 1; var string : like "hello world") is
	var x = 5;
end;