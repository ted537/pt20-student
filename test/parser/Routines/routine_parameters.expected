 .sProgram
   .sIdentifier
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
    .sProcedure
     .sIdentifier
        .sIdentifier
         .sLike
         .sInteger
         .sIdentifier
        .sVar
         .sLike
         .sStringLiteral
    % .sNewLine
    .sParmEnd
     .sBegin
       .sVar
        .sIdentifier
        .sInitialValue
            .sInteger
       % .sNewLine
     .sEnd
  .sEnd
